src/Features/VisualBasic/Portable/PullMemberUp/VisualBasicMembersPullerService.vb

```diff
@@ -0,0 +1,26 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Composition
+Imports Microsoft.CodeAnalysis.Host.Mef
+Imports Microsoft.CodeAnalysis.Simplification
+Imports Microsoft.CodeAnalysis.VisualBasic
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.CodeRefactorings.PullMemberUp
+    <ExportLanguageService(GetType(IMembersPullerService), LanguageNames.VisualBasic), [Shared]>
+    Friend Class ViualBasicMembersPullerService
+        Inherits AbstractMembersPullerService
+
+        <ImportingConstructor>
+        <Obsolete(MefConstruction.ImportingConstructorMessage, True)>
+        Public Sub New()
+        End Sub
+
+        Protected Overrides Function FilterType(node As SyntaxNode) As Boolean
+            Return TypeOf node Is IdentifierNameSyntax Or TypeOf node Is ObjectCreationExpressionSyntax
+        End Function
```

Comment:
we don't need a special service for thsi.  these services can be removed and ISyntaxFactsService or ISyntaxKindsService can be used instead from the original code to make this determination for both C# and VB.